vue.config.js

```diff
@@ -14,10 +13,6 @@ const aboutText2 = (sbcName && sbcVersion) ? `${sbcName} v${sbcVersion}` : ''
 module.exports = {
   configureWebpack: {
     plugins: [
-      // this is needed to remove existing service workers on users' systems
-      // ref: https://www.npmjs.com/package/webpack-remove-serviceworker-plugin
-      // ref: https://github.com/NekR/self-destroying-sw/tree/master/packages/webpack-remove-serviceworker-plugin
-      new RemoveServiceWorkerPlugin({ filename: 'service-worker.js' }),
```

Comment:
Reason for removing this: https://github.com/bcgov/entity/issues/16170

Reply:
You may be able to delete src/regitsterServiceWorker.ts.

See https://github.com/bcgov/business-filings-ui/pull/266 for the changes that were done to implement this feature that you are now removing.

Reply:
If the service worker is stuck - There's a certain header you can setup on NGINX that gets it unstuck. We had to do this when we loaded on a bad version of the service worker that turfed some clients.

Reply:
That's nice to know. Instead, we used this plugin for over a year to remove the SW.

Reply:
Yeah the plugin wouldn't work if the service worker is toast and errors in javascript - Kinda last ditch effort we had to do with Director Search

Reply:
Sev and Travis, I removed the `registerServiceWorker.ts` file with it's respective import in `main.ts` in my latest commit. Please let me know if you think that's OK to do.

Reply:
I think so... ? Try `npm run build`. Maybe we won't know for sure until it's deployed to Dev.

Reply:
I did try a build, everything looks OK. I also tested in local. Everything seems to be running normally. Okay then, hopefully all goes smoothly when I merge. Thanks!